packages/myst-cli/src/build/jats/single.ts

```diff
@@ -29,6 +35,9 @@ export async function runJatsExport(
   const toc = tic();
   const { output, articles, sub_articles } = exportOptions;
   // At this point, export options are resolved to contain one-and-only-one article
+  if (articles.length > 1) {
+    throw new Error('When specifying a named output for export, you must list exactly one file.');
+  }
```

Comment:
The verbal asserts imply `articles.length === 1`, but the later code permits `.length === 0`, so I've imposed this.